maps/~mapsystem/maps.dm

```diff
@@ -128,17 +128,19 @@ var/global/const/MAP_HAS_RANK = 2		//Rank system, also togglable
 	var/default_species = SPECIES_HUMAN
 
 	var/list/available_cultural_info = list(
-		TAG_HOMEWORLD = list(/decl/cultural_info/location/other),
-		TAG_FACTION =   list(/decl/cultural_info/faction/other),
-		TAG_CULTURE =   list(/decl/cultural_info/culture/other),
-		TAG_RELIGION =  list(/decl/cultural_info/religion/other)
+		TAG_HOMEWORLD =   list(/decl/cultural_info/location/other),
+		TAG_FACTION =     list(/decl/cultural_info/faction/other),
+		TAG_CULTURE =     list(/decl/cultural_info/culture/other),
+		TAG_RELIGION =    list(/decl/cultural_info/religion/other),
+		TAG_CITIZENSHIP = list(/decl/cultural_info/citizenship/other)
```

Comment:
As with the other PR, we ought to set this on a specific map, but doing it here creates blanket baseline so most other maps should compile w/ the changed ALL_CULTURAL_DATUMS list

Reply:
Updated to only affect tradeship, will need to copy that to other maps as needed